printable-recipe-generator/src/main.ts

```diff
@@ -3,8 +3,63 @@ import * as process from 'node:process';
 import path from 'path';
 import type { Data } from 'ejs';
 import { render as renderTemplate } from 'ejs';
+import fetch from 'node-fetch';
+
+//load Contributors
+async function getChefs(): Promise<unknown> {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call -- response is expected to be JSON, safe to assign
+	const resp = await fetch(
+		'https://recipes.guardianapis.com/v2/contributors.json',
+	);
+	try {
```

Comment:
Personally I wouldn't try/catch here. Instead, I'd allow `resp.json()` to throw an exception and catch it in the caller, something like:

```
let chefs:Record<string, ChefData>|null = null;
try {
  chefs = await loadChefs();
} catch(err) {
  console.error("Could not load chefs: ", err);
}
```

Crucially, you need to remember that `chefs` is now nullable (made explicit by the typing in my example).  Typing it in this way will cause Typescript to remind you of the fact that it _is_ nullable - lines like `console.log(chefs['profile/yotamottolenghi'].webTitle);` will definitely crash if `chefs` is `null` and trying to reference it in the template will likely also cause a carsh

Reply:
yes changes as mentioned above, resolves this